simpleflow/execute.py

```diff
@@ -378,13 +410,31 @@ def make_callable(funcname):
             result = callable_(*args, **kwargs)
     except Exception as err:
         logger.error('Exception: {}'.format(err))
-        # Use base64 encoding to avoid carriage returns and special characters.
-        # FIXME change this: brittle, missing traceback
-        encoded_err = base64.b64encode(pickle.dumps(err))
+        exc_type, exc_value, exc_traceback = sys.exc_info()
+        tb = traceback.format_tb(exc_traceback)
+        details = json_dumps(
```

Comment:
Use `swf.format.encode()` or equivalent helpers instead (translate jumbo fields automatically, so we can have traceback larger than 32k yay!)